en/flight_controller/modalai_fc_v1.md

```diff
@@ -0,0 +1,82 @@
+# ModaAI Flight Core v1
+
+The ModalAI Flight Core is a [Pixhawk Series](../flight_controller/pixhawk_series.md) derived flight controller, made in the USA. The Flight Core can be paired with [ModalAI VOXL](https://docs.modalai.com/voxl-datasheet/) for obstacle avoidance and GPS-denied navigation, or used independently as a standalone flight controller.
+
+![FlightCoreV1](../../assets/hardware/hardware-modalai-fc-v1.jpg)
+
+## Quick Summary
+
+The main hardware documentation can be found [here].(https://docs.modalai.com/flight-core-datasheet/)
+
+## Specification
+
+| Feature          | Details |
```

Comment:
Can we add physical dimension? i.e. weight, size.

Reply:
will do now